paywall/src/middlewares/currencyConversionMiddleware.js

```diff
@@ -4,6 +4,10 @@
 import axios from 'axios'
```

Comment:
I am confused because I thought you wanted to remove the polling from services to make it easier to re-use the services inside the pawyall application. Why change the middleware inside the experiment?